packages/ember-views/lib/system/render_buffer.js

```diff
@@ -514,7 +514,7 @@ RenderBuffer.prototype = {
     var content = this.innerContent();
     // No content means a text node buffer, with the content
     // in _element. Ember._BoundView is an example.
-    if (content === null) {
+    if (content === null && typeof(content) === "undefined") {
```

Comment:
This doesn't make sense. Something can't both be `null` *and* `undefined`, which is what this is testing for.